### DISCUSSION TITLE
Issue with Non-Receipt of Contributions after Commits on the Master Branch

### DISCUSSION THREAD

**Discussion by ManuYao** [community/community#73103] [Repositories]
### Select Topic Area

Bug

### Body

**Dear community,**

I hope this message finds you all well. I am reaching out to you today because I am facing a persistent issue that is impacting my motivation and efficiency in using Git.

**Every time I commit changes to the master branch, unfortunately, I do not receive any contributions added to my list. This problem has been ongoing for a while, and despite my efforts to resolve it, I have not yet found a solution.**

I want to clarify that the only time I receive contributions is during the creation of a new repository. I have explored various online resources, browsed through different websites, but I have not been able to identify the underlying cause of this malfunction.

**I have also checked various settings, ensuring, among other things, that my repositories are not set to private. Everything seems to be correctly configured to receive contributions, but unfortunately, it is not happening.**

I am reaching out to seek your expertise and experiences to help me resolve this issue. If any of you have encountered a similar situation or can provide useful advice, I would be extremely grateful.

Together, as a community, I am confident that we can find a solution to this problem. Thank you in advance for your valuable assistance.

**Comment by ktauchathuranga** [comment]
you have to make sure your local Git commit email is connected to your account.

refer this [Docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#common-reasons-that-contributions-are-not-counted) hope this helps.
happy coding 🖤

**Reply by ktauchathuranga** [reply]
refer [this](https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/) to change your Git email or the name
🖤

**Reply by ManuYao** [reply]
thx
